tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -112,10 +137,8 @@ export class Ide {
         await this.driverHelper.waitAndClick(By.xpath(yesButtonLocator));
     }
 
-    async waitWorkspaceAndIde(workspaceNamespace: string,
-        workspaceName: string,
-        timeout: number = TestConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT) {
-
+    async waitWorkspaceAndIde(workspaceNamespace: string, workspaceName: string, timeout: number = TestConstants.TS_SELENIUM_LOAD_PAGE_TIMEOUT) {
+        // why are there unused arguments in this method?
```

Comment:
We should consider refactoring this method. It's tied to HappyPath tests as well.

Reply:
@Ohrimenko1988 ^